src/typings/common.ts

```diff
@@ -0,0 +1,32 @@
+/*
+ * Please refer to the terms of the license agreement.
```

Comment:
Common typings. The types file will likely be extended with more commonly used props.